packages/ord-connect/src/lib/signMessage.ts

```diff
@@ -0,0 +1,40 @@
+import { ordit } from "@sadoprotocol/ordit-sdk";
+import { Network, Wallet } from "../providers/OrdContext.tsx";
+
+interface SignMessageParams {
+  message: string;
+  wallet: Wallet;
+  address: string | Record<string, string>;
+  network: Network;
+}
+
+export default async function signMessage({
+  message,
+  wallet,
+  address,
+  network,
+}: SignMessageParams) {
+  if (wallet === Wallet.UNISAT) {
+    const signedMessage = await ordit.unisat.signMessage(message);
+    return signedMessage.base64;
+  }
+
+  if (wallet === Wallet.XVERSE) {
+    let xverseAddress = address as string;
+    if (typeof address === "object") {
+      xverseAddress = address.ordinals;
+    }
```

Comment:
let's just enforce address to be passed in as a single string. if object, return error.

Reply:
updated